frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx

```diff
@@ -187,6 +189,18 @@ const InfraEnvsTable: React.FC<InfraEnvsTableProps> = ({ infraEnvs, agents }) =>
                             )
                         },
                     },
+                    {
+                        header: t('infraEnv.tableHeader.creationDate'),
+                        sort: (a: InfraEnvK8sResource, b: InfraEnvK8sResource) => {
```

Comment:
IIUC this should work as well:
```
sort: 'metadata.creationTimestamp',
```

Reply:
To sort correctky date fields we need to do this:
       const dateTimeCellA = getDateTimeCell(a.metadata?.creationTimestamp ? new Date(a.metadata?.creationTimestamp).toString() : '-');
                            const dateTimeCellB = getDateTimeCell(b.metadata?.creationTimestamp ? new Date(b.metadata?.creationTimestamp).toString() : '-'); 
                            return compareStrings(((dateTimeCellA.sortableValue == 0) ? '':dateTimeCellA.sortableValue.toString()), ((dateTimeCellB.sortableValue == 0) ? '':dateTimeCellB.sortableValue.toString()))